src/gwt/panmirror/src/editor/src/behaviors/insert_symbol/insert_symbol-grid.tsx

```diff
@@ -116,6 +116,7 @@ const SymbolCharacterGrid = React.forwardRef<any, CharacterGridProps>((props, re
       previewTimer.current = window.setTimeout(() => {
         setShowPreview(true);
       }, kWaitToShowPreviewMs);
+      return () => { window.clearTimeout(previewTimer.current); };
```

Comment:
Will this work correctly even if `previewTimer.current` is unset/null?

Reply:
Yep - if it isn't initialized It will be undefined (clearTimeout is ok with this).